src/plugin-toggler.php

```diff
@@ -178,8 +178,10 @@ public function ajax_toggle_plugin() {
 
 		// If nonce is valid.
 		if ( $this->verify_nonce() ) {
-			$group  = \filter_input( \INPUT_GET, 'group' );
-			$plugin = \filter_input( \INPUT_GET, 'plugin' );
+			// phpcs:ignore WordPress.Security.NonceVerification.Recommended -- The nonce is verified above.
+			$group = ( isset( $_GET['group'] ) ) ? \sanitize_text_field( \wp_unslash( $_GET['group'] ) ) : null;
+			// phpcs:ignore WordPress.Security.NonceVerification.Recommended -- The nonce is verified above.
+			$plugin = ( isset( $_GET['plugin'] ) ) ? \sanitize_text_field( \wp_unslash( $_GET['plugin'] ) ) : null;
```

Comment:
Same as above, about the `is_string`. As well as using `null` as fallback, that does not seem compatible with the methods they are passed too.